<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>175</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioAdding">
         <property name="text">
          <string>Adding mode</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioEditing">
         <property name="text">
          <string>Editing mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioAlgorithm">
         <property name="text">
          <string>Algorithm mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClearMarking">
         <property name="text">
          <string>Clear marking</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonInformation">
         <property name="text">
          <string>Information</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonEding">
         <property name="text">
          <string>Eding</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Convert from file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClearMarking_4">
         <property name="text">
          <string>Convert to file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRandomGraph">
         <property name="text">
          <string>Make random graph</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkDefaultWeight">
         <property name="text">
          <string>Default weigh</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinDefaultWeight">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="listView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart_algorithm">
    <property name="title">
     <string>Start algorithm</string>
    </property>
    <addaction name="actionAlgo_1"/>
    <addaction name="actionAlgo_2"/>
    <addaction name="actionAlgo_3"/>
    <addaction name="actionAlgo_4"/>
    <addaction name="actionAlgo_5"/>
    <addaction name="actionAlgo_6"/>
    <addaction name="actionAlgo_7"/>
   </widget>
   <addaction name="menuStart_algorithm"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Ñ–</string>
   </property>
  </action>
  <action name="actionAlgo_1">
   <property name="text">
    <string>Algo-1</string>
   </property>
  </action>
  <action name="actionAlgo_2">
   <property name="text">
    <string>Algo-2</string>
   </property>
  </action>
  <action name="actionAlgo_3">
   <property name="text">
    <string>Algo-3</string>
   </property>
  </action>
  <action name="actionAlgo_4">
   <property name="text">
    <string>Algo-4</string>
   </property>
  </action>
  <action name="actionAlgo_5">
   <property name="text">
    <string>Algo-5</string>
   </property>
  </action>
  <action name="actionAlgo_6">
   <property name="text">
    <string>Algo-6</string>
   </property>
  </action>
  <action name="actionAlgo_7">
   <property name="text">
    <string>Algo-7</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>radioAdding</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonClearMarking</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioEditing</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonClearMarking</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
